source/fundamentals/data-formats/documents.txt

```diff
@@ -15,15 +15,14 @@ Overview
 --------
 
 A MongoDB **document** is a data structure that contains key/value pairs in
-binary JSON (BSON) format. The document is the base unit of the MongoDB
-data hierarchy: each document belongs to a single collection
+binary JSON (BSON) format. Each document belongs to a single collection
```

Comment:
I still think this is misleading since we use documents for queries, commands, etc.  Those don't belong to a collection.

Reply:
Ok thanks, I'll update the sentence. My intent is to build a little more context beyond the first sentence without having the user follow the link. If you think that's extraneous or if there's some other essential info you'd prefer to include instead, let me know.